src/Cart/CartMobile.js

```diff
@@ -0,0 +1,43 @@
+import Product from "../Product/Product";
+import { useCart } from "./CartContext";
+import { Link } from "react-router-dom";
+
+const CartMobile = () => {
+  const { cart, total, items } = useCart();
+
+  if (items === 0) {
+    return (
+      <div className="flex-c justify-c h-full-vp">
+        <span className="x-large">Your cart is empty</span>
+        <Link to="/">
+          <button className="btn btn-classic shadow medium mt-m">
+            Go Shopping
+          </button>
+        </Link>
+      </div>
+    );
+  } else {
+    return (
+      <div>
+        <h2 className="m-m">Cart</h2>
+        <div className="cart-products">
+          {cart.map((product) => {
+            return (
+              <>
+                <Product key={product._id} product={product} cartView={true} />

```

Comment:
liked that you are utilizing product component in cart. makes the code more readable